libraries/apollo-ast/src/main/kotlin/com/apollographql/apollo3/ast/introspection/IntrospectionSchema.kt

```diff
@@ -13,30 +13,31 @@ import okio.source
 import java.io.File
 
 @Serializable
-private data class IntrospectionSchemaEnvelope(
+private class IntrospectionSchemaEnvelope(
```

Comment:
Might be worth doing a separate PR for this? I think we'll want `equals`/`hashCode`/`toString` implementations to keep working? Also we can then do it for `gql.kt` at the same time

Reply:
`toString()` is nice for debugging. Not sure about equals/hashCode